Append lines that start with a hyphen to the current content during parsing

ubermichael opened the pull request:
The MIME parser processes content line by line. If a line starts with a hyphen (-) it is examined as a potential MIME boundary. If it isn't a boundary, the parser moves on to the next line. It should append the line to the content being processed.

Fixes BeSimple/BeSimpleSoap#87

ubermichael committed:
Append lines that start with a hyphen to the current content during
MIME parsing.

Fixes BeSimple/BeSimpleSoap#87

ubermichael committed:
Fix some namespace conflicts.

guilhermeadc commented:
This fix needs to be applied as soon as possible. This problem is corrupting various files, mostly PDFs, received by the SOAP + MTOM.

ubermichael committed:
Update the composer dependencies.

I have no idea if this will work for any purposes. The test suite
started off broken as shit, but it's still broken in the same way so
maybe with luck this will work.
